apps/hubble/src/addon/src/store/tag_store.rs

```diff
@@ -188,7 +188,7 @@ impl TagStoreDef {
 
         // VIC-TODO: blake3_20 hash here?
         // Ok((by_target_key, tag_body.r#type as u8))
-        Ok((by_target_key, 0 as u8))
+        Ok((by_target_key, tag_body.r#type.as_bytes().to_vec()))
```

Comment:
Interesting. Nice that Vic left some TODOs around here too.
So we're now returning `r#type` as a secondary index (the tag value rather than `0`) and using it as part of the `txn.put` upstream

Reply:
yup, Vic had a thought of using a hash here. I just converted to bytes similar to what link_store does.